modules/apps/document-library/document-library-web/src/main/resources/content/Language.properties

```diff
@@ -74,6 +74,7 @@ get-url-or-webdav-url=Get {0}URL{1} or {2}WebDAV URL{3}.
 go-to-folder=Go to Folder
```

Comment:
Did you run buildLang? I don't see the key propagated to other files.

Reply:
![](https://media0.giphy.com/media/AjYsTtVxEEBPO/giphy.gif)
thanks!